source/jormungandr/tests/disruptions_tests.py

```diff
@@ -871,3 +871,57 @@ def test_line_reports_with_since_until_value_not_valid(self):
         )
         assert code == 404
         assert response['error']['message'] == 'invalid filtering period (since > until)'
+
+    def test_line_reports_with_uri_filter(self):
+        # Without filter
+        response = self.query_region("line_reports?_current_datetime=20170101T120000")
+        line_reports = get_not_null(response, 'line_reports')
+        assert len(line_reports) == 2
+        # line:1
+        assert line_reports[0]['line']['id'] == 'line:1'
+        assert len(line_reports[0]['line']['links']) == 2
+        assert line_reports[0]['line']['links'][0]['id'] == 'line_section_on_line_1'
+        assert line_reports[0]['line']['links'][1]['id'] == 'line_section_on_line_1_other_effect'
```

Comment:
Updated the suggested test a bit, to check that we also don't have pt_objects without links.

Reply:
Yes, it is true :+1: 

We didn't see the dead link inside `line_reports->pt_object->embedded_type->links->associated_disruptions`.
for my part, I only check `line_reports->lines->links->associated_disruptions`.
I'll add this check inside the test, and the PR will become red because of it.

Nevertheless, it not change the fact, that I don't have, for the moment, enough legacy knowledge on this part. As I said into the ticket and during grooming, If you detect some part of code that is culprit, fell free to propose a change. I suppose we will talk all together about that next time.